core/new-gui/src/app/workspace/component/navigation/navigation.component.ts

```diff
@@ -371,4 +391,36 @@ export class NavigationComponent implements OnInit {
       });
   }
 
+  handleDisableOperatorIconChange() {
+    Observable.merge(
+      this.workflowActionService.getJointGraphWrapper().getJointOperatorHighlightStream(),
+      this.workflowActionService.getJointGraphWrapper().getJointOperatorUnhighlightStream(),
+      this.workflowActionService.getJointGraphWrapper().getJointGroupHighlightStream(),
+      this.workflowActionService.getJointGraphWrapper().getJointGroupUnhighlightStream(),
+      this.workflowActionService.getTexeraGraph().getDisabledOperatorsChangedStream(),
+    ).subscribe(event => {
+      this.isDisableOperator = this.calculateIsDisableOperator();
+      this.isDisableOperatorClickable = this.effectivelyHighlightedOperators().length !== 0;
+    });
+  }
+
+  calculateIsDisableOperator(): boolean {
```

Comment:
it will be good to give some docstring/comments. The function name is somewhat confusing.

Reply:
added